Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandRenamePlayerHouse.java

```diff
@@ -34,9 +35,10 @@ public void run(Map<String, String> parameters) throws UserErrorMessage {
 		CachedEntity character = db.getCurrentCharacter();
 		Key locationKey = (Key)character.getProperty("locationKey");
 		CachedEntity location = db.getEntity(locationKey);
+		CachedEntity owner = db.getEntity((Key)location.getProperty("ownerKey"));
```

Comment:
I'm pretty sure you don't need the owner/user entity at all. You just need the key, which you get from the getProperty() call here. So the db.getEntity() call is hitting the DB unnecessarily.